frontend/src/components/NewItemCard.tsx

```diff
@@ -0,0 +1,91 @@
+import {Item} from "../models/Item.ts";
+import {FormEvent, useEffect, useState} from "react";
+import axios from "axios";
+import "./NewItemCard.css";
+
+export default function NewItemCard() {
+
+    const itemData: Item = {
+        name: "item ame",
+        img: "item img_url",
+        description: "item description",
+        category: "TOOL",
+        status: "TO_SELL"
+    };
+
+    const [item, setItem] = useState<Item>(itemData)
+
+    const [itemName, setItemName] = useState("");
+    const [itemImg, setItemImg] = useState("");
+    const [itemDescription, setItemDescription] = useState("");
+    const [itemCategory, setItemCategory] = useState("");
+    const [itemStatus, setItemStatus] = useState("");
+
+
+
+    function addItem(event: FormEvent<HTMLFormElement>) {
+        event.preventDefault()
+        axios.post("/api/item", itemData).then(
+            (response)=>{
+                setItem(response.data)
+            }
+        )
+        console.log({itemName,itemImg,itemDescription,itemCategory,itemStatus})
+    }
```

Comment:
Error handling missing, when addItem is invalid.
Delete console log or log the Error